meson.build

```diff
@@ -40,15 +43,92 @@ postgresql = declare_dependency(
   ],
 )
 
+pgrn_dependencies = [postgresql, groonga, libm]
+
+msgpack = dependency('msgpack-c', 'msgpack', required: get_option('message_pack'))
+if msgpack.found()
+  pgrn_msgpack = declare_dependency(
+    compile_args: ['-DPGRN_HAVE_MSGPACK'],
+    dependencies: [msgpack]
+  )
+  pgrn_dependencies += pgrn_msgpack
+  message('MessagePack: system')
```

Comment:
We don't need this because Meson shows it: https://github.com/pgroonga/pgroonga/actions/runs/15867044950/job/44735803391?pr=750#step:7:27

```text
Run-time dependency msgpack found: YES 4.0.0
```

```suggestion
```